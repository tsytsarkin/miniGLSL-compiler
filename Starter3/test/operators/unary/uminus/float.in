{ float y; float x; y = -x; }
