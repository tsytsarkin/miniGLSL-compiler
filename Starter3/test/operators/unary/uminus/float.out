  1: { float x; y = -x; }
(SCOPE (DECLARATIONS (DECLARATION x float ) ) (STATEMENTS (ASSIGN unknown y (UNARY float - x ) ) ) ) 
Failed to compile
